server/datastore/mysql/software_test.go

```diff
@@ -3669,16 +3674,16 @@ func testListHostSoftware(t *testing.T, ds *Datastore) {
 	vpp1TmCmdUUID := createVPPAppInstallRequest(t, ds, tmHost, vpp1, user.ID)
 	require.NotEmpty(t, vpp1TmCmdUUID)
 
-	expected["vpp1apps"] = fleet.HostSoftwareWithInstaller{
-		Name:        "vpp1",
-		Source:      "apps",
+	expected["vpp1"] = fleet.HostSoftwareWithInstaller{
+		Name: "vpp1",
+		// Source:      "apps",
```

Comment:
I'm not sure I understand what's going on here 🤔 did something change under the hood?

Reply:
We set the `source` column to be empty when we insert into the software_titles table (I should have removed the commented code tho, was just confirming that it works)

Reply:
I completely forgot: Gabe raised the issue that an empty `source` field leads to issues in the UI. I think we should add a `vpp_apps` type to the `source` field for the software title. WDYT @roperzh ? 

If that's the case, I'll need to update this PR with the correct source

Reply:
did you folks discuss if we could use any of the sources that we have? I think of source as the "local-level" source. I believe VPP apps will probably be reported as just "apps" by osquery

Reply:
sounds like we decided `apps` [in Slack](https://fleetdm.slack.com/archives/C03C41L5YEL/p1721405629659889) just now. I'll update the PR.